### PR TITLE
Add recipe for fluffydiscord/roadrunner-symfony-bundle

### PR DESCRIPTION
**Pull request by FluffyDiscord**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     | https://packagist.org/packages/fluffydiscord/roadrunner-symfony-bundle

### CODE REVIEW

**Comment by FluffyDiscord** [comment]
The run updated recipes Github action is failing because of unsupported PHP version. Am I required to downgrade my bundle to support php 7.4 and 8.1?

File: fluffydiscord/roadrunner-symfony-bundle/2.3/config/packages/fluffy_discord_road_runner.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+framework:
+    session:
+        # Fix for Symfony not handling sessions properly in worker mode
+        storage_factory_id: FluffyDiscord\RoadRunnerBundle\Session\WorkerSessionStorageFactory
```
Comment (fabpot) [inline fluffydiscord/roadrunner-symfony-bundle/2.3/config/packages/fluffy_discord_road_runner.yaml:4]: I don't think this can be part of the recipe.

**Comment by diimpp** [comment]
@FluffyDiscord Hi, do you plan to resolve review's comments?

> The run updated recipes Github action is failing because of unsupported PHP version. Am I required to downgrade my bundle to support php 7.4 and 8.1?

No, it's just how CI configured, PR can be accepted even without passing tests due version mismatch.

**Comment by FluffyDiscord** [comment]
> @FluffyDiscord Hi, do you plan to resolve review's comments?
> 
> > The run updated recipes Github action is failing because of unsupported PHP version. Am I required to downgrade my bundle to support php 7.4 and 8.1?
> 
> No, it's just how CI is configured, PR can be accepted even without passing tests due version mismatch.

Sorry for the delay. I will look at it tomorrow.

File: fluffydiscord/roadrunner-symfony-bundle/2.3/config/packages/fluffy_discord_road_runner.yaml (hunk 1 above)
Comment (FluffyDiscord) [reply fluffydiscord/roadrunner-symfony-bundle/2.3/config/packages/fluffy_discord_road_runner.yaml]: removed

**Review by fabpot — APPROVED** [review]

**Pull request merged**